docs/api.md

```diff
@@ -1704,7 +1704,8 @@ await page.evaluate(() => matchMedia('print').matches);
   - `latency` <[number]> Latency (ms), `0` to disable
 - returns: <[Promise]>
 
-> **NOTE** This does not affect WebSockets and WebRTC PeerConnections (see https://crbug.com/563644)
+> **NOTE** This does not affect WebSockets and WebRTC PeerConnections (see https://crbug.com/563644). To set the page offline, you can use [page.setOfflineMode(enabled)](#pagesetofflinemodeenabled).
```

Comment:
This part is not auto-generated right?

Reply:
No, I added it manually. Is `api.md` now generated from code?

Reply:
I am not sure, I thought so but I might be wrong :D

Reply:
The same strings can be found in `Page.ts` - I updated them too to keep them in sync.
@mathiasbynens Do we already generate documentation from source automatically?